app/mailers/application_mailer.rb

```diff
@@ -1,17 +1,26 @@
-# Application Mailer
+# Application mailer base class (NB: user mailer does not inherit from this)
```

Comment:
> Application mailer base class (NB: user mailer does not inherit from this)

(Apologies if you're already aware of this and its just not desirable.) 

Devise's mailer's parent be configured to be `ApplicationMailer` using the `parent_mailer` config. See the related code here:

https://github.com/heartcombo/devise/blob/64863513506f4f2913510a050194d158c979eacc/app/mailers/devise/mailer.rb#L4

Reply:
Ah! No, I was cobbling bits together from a few different blog posts and none of them mentioned that. Although ...